lib/hooks.ts

```diff
@@ -75,3 +75,20 @@ export const useOrder = (orderId: number) => {
         error,
     };
 }
+
+export const useShippingAndProductsOrder = (orderId: number) => {
```

Comment:
🍹 Does this name make sense? Since it's **only** fetching Shipping Info and Products could we rename this to `useOrderShippingAndProductInfo` or something of that sorts?